### PR TITLE
added empty mode (isolated net)

### PR DESCRIPTION
**Pull request by IncredibleRichie**
closes https://github.com/stackhpc/ansible-role-libvirt-host/issues/38

### CODE REVIEW

File: templates/network.xml.j2 [hunk 1]
```diff
@@ -1,29 +1,44 @@
 <network {% if item.ipv6 is defined %}ipv6='yes'{% endif %} connections='1'>
   <name>{{ item.name }}</name>
-  <forward mode='{{ item.mode }}'/>
+  {% if item.mode is defined %}<forward mode='{{ item.mode }}'/>{% endif %}
   <bridge name='{{ item.bridge }}'/>
-{% if item.mode == 'route' or item.mode == 'nat' %}
+{% if item.mode is defined %}
```
Comment (markgoddard) [inline templates/network.xml.j2:5]: I don't think we need this condition - it seems many of these things are still supported: https://libvirt.org/formatnetwork.html#isolated-network-config

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: templates/network.xml.j2 (hunk 1 above)
Comment (IncredibleRichie) [reply templates/network.xml.j2:5]: You´re right, i´ve changed it.

File: templates/network.xml.j2 [hunk 2]
```diff
@@ -25,5 +25,18 @@
     <route family="ipv6" address="{{ route.address }}" prefix="{{ route.prefix }}" gateway="{{ route.gateway }}" {% if route.metric is defined %} metric='{{ route.metric }}' {% endif %}/>
   {% endfor %}
   {% endif %}
+{% else %}
+  <domain name='{{ item.domain|default(item.name) }}'/>
```
[the remark this answers is not in the document]
Comment (IncredibleRichie) [reply templates/network.xml.j2]: yes that saves some lines

**Review by markgoddard — APPROVED** [review]
Thanks

**Pull request merged**